README.md

```diff
@@ -11,7 +11,7 @@ npm install jsbi --save
 ## Usage
 
 ```js
-import JSBI from './jsbi.mjs';
```

Comment:
Can you elaborate on this “fix”? In a browser, `import JSBI from './jsbi.mjs';` is how you’d have to do it, unless you’re using import maps.